src/SFA.DAS.Apprenticeships.Web/Controllers/ChangeOfPriceProviderController.cs

```diff
@@ -188,8 +231,8 @@ public async Task<IActionResult> CancelPriceChange(long ukprn, string apprentice
 			return pendingPriceChange;
 		}
 
-		//  If other provider endpoints use the same route values, this could be refactored to take an interface/abstract class instead of CreateChangeOfPriceModel
-		private void PopulateRouteValues(IRouteValuesProvider model)
+        //  If other provider endpoints use the same route values, this could be refactored to take an interface/abstract class instead of CreateChangeOfPriceModel
```

Comment:
My bad, should have removed this comment when I refactored the endpoint to use an interface, can you remove the comment